### PR TITLE
Add signing functions for timestamp support

### PR DESCRIPTION
**Pull request by a5i**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module github.com/unidoc/pkcs7
+
+go 1.14
```
Comment (gunnsth) [inline go.mod]: does this work on earlier go versions? would be good if we can get this down to 1.11

**Review by gunnsth — COMMENTED** [review]
Looks good, 1 minor comment.

File: go.mod (hunk 1 above)
Comment (a5i) [reply go.mod]: I tested with 1.11 version it works.
Updated to 1.11.

**Review by gunnsth — APPROVED** [review]
LGTM

**Pull request merged**